newIDE/app/src/Export/Builds/BuildsList.js

```diff
@@ -20,63 +23,155 @@ type Props = {|
   loadBuilds: () => void,
   game: Game,
   onGameUpdated?: Game => void,
+  onBuildUpdated: Build => void,
+  onBuildDeleted: Build => void,
 |};
 
+type BuildFilter = BuildType | 'all-build';
+
+const selectOptionsArray: Array<{
+  key: BuildFilter,
+  value: BuildFilter,
+  primaryText: React.Node,
+}> = [
+  {
+    key: 'all-build',
+    value: 'all-build',
+    primaryText: t`All builds`,
+  },
+
+  {
+    key: 'web-build',
+    value: 'web-build',
+    primaryText: t`Web builds`,
+  },
+
+  {
+    key: 'cordova-build',
+    value: 'cordova-build',
+    primaryText: t`Mobile builds`,
+  },
+
+  {
+    key: 'electron-build',
+    value: 'electron-build',
+    primaryText: t`Desktop builds`,
+  },
+];
+
+const filterBuilds = (builds: ?Array<Build>, buildFilter: BuildFilter) => {
+  if (!builds) return;
+  switch (buildFilter) {
+    case 'web-build':
+      return builds.filter(build => build.type === 'web-build');
+    case 'cordova-build':
+      return builds.filter(build => build.type === 'cordova-build');
+    case 'electron-build':
+      return builds.filter(build => build.type === 'electron-build');
+    default:
+      return builds;
+  }
+};
+
+const emptyBuildMessage = {
+  'web-build': <Trans>You don't have any web builds for this game.</Trans>,
+  'cordova-build': (
+    <Trans>You don't have any mobile builds for this game.</Trans>
+  ),
+  'electron-build': (
+    <Trans>You don't have any desktop builds for this game.</Trans>
+  ),
+  'all-build': <Trans>You don't have any builds for this game.</Trans>,
+};
+
 const BuildsList = ({
   builds,
   authenticatedUser,
   error,
   loadBuilds,
   game,
   onGameUpdated,
+  onBuildUpdated,
+  onBuildDeleted,
 }: Props) => {
   const [gameUpdating, setGameUpdating] = React.useState(false);
+  const [buildFilter, setBuildFilter] = React.useState<BuildFilter>(
+    'all-build'
+  );
+  const displayedBuilds = filterBuilds(builds, buildFilter);
+
   return (
     <Column noMargin expand>
-      <Line>
+      <ResponsiveLineStackLayout justifyContent="space-between">
         <Column>
-          <EmptyMessage>
+          <EmptyMessage messageStyle={{ textAlign: 'left' }}>
             <Trans>
-              This is the list of builds that you've done for this game. Note
-              that builds for mobile and desktop are available for 7 days, after
-              which they are removed.
+              This is the list of builds that you've done for this game. <br />
+              Note that builds for mobile and desktop are available for 7 days,
+              after which they are removed.
             </Trans>
           </EmptyMessage>
         </Column>
-      </Line>
+        <Column>
+          <div style={{ maxWidth: '350' }}>
```

Comment:
```suggestion
          <div style={{ maxWidth: 350 }}>
```

Can you comment, or better, give a name to 350 in the form of a constant?